docs/swagger/api.yaml

```diff
@@ -0,0 +1,366 @@
+swagger: "2.0"
+info:
+  description: "Squash debugger api."
+  version: "2.0.0"
+  title: "Squash Server"
+  contact:
+    email: "apiteam@solo.io"
```

Comment:
is this an email we monitor?

Reply:
actually not sure what that is; I should probably check